CONTRIBUTING.md

```diff
@@ -1,26 +1,53 @@
-# Contributing / Development workflow
+# Contributing
+
+So excited to have you here! If you want **any** guidance whatsoever with contributing to tRPC, don't hesitate to reach out on [Discord](https://trpc.io/discord)!
+
+
+## Development workflow
 
 ```bash
 git clone git@github.com:trpc/trpc.git
 cd trpc
 yarn
 ```
 
-## Hacking around with it
-
+### Get it running
 
-In one terminal, will run `preconstruct watch` in parallel which builds all `packages/*` on change:
+**Terminal 1:**
 
 ```bash
+# in project root directory
 yarn dev
 ```
 
+This will start a watcher in parallel which builds all `packages/*` on any file change.
+
+**Terminal 2:**
+
 In another terminal, you can for instance navigate to `examples/next-prisma-starter` and run `yarn dev` & it will update whenever code is changed in the packages.
 
-## Testing
+### Testing
+
+> Note: you will want to have `yarn dev` running in parallel in another terminal
```

Comment:
Is this needed? Don't the tests run the actual source TS files?

Reply:
If they're setup with mock properly yes, unsure if all test files import the _packages thing

Reply:
If they're setup with mock properly yes, unsure if all test files import the _packages thing

Reply:
I think we just import the files directly instead of using the `@trpc/client`, `@trpc/react imports. [Example](https://github.com/trpc/trpc/blob/c4138897578949d910dd0bce9e1471a6ae73137e/packages/server/test/errors.test.ts#L9)

Reply:
Yeah, but you can import something from the client that then uses the `@trpc/*` which then requires the mock to be setup first ☺

Reply:
I'll merge it as is for now and we can revisit later!